packages/tokens/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@canonical/tokens",
+  "description": "Canonical design tokens: primitives and mode mappings for use with Style Dictionary",
+  "version": "0.9.0-experimental.1",
+  "main": "src/index.css",
+  "files": ["src"],
```

Should this be `dist`? 
```suggestion
  "files": ["dist"]
```